ontologies/works.ttl

```diff
@@ -70,13 +80,27 @@ ww:hasItem rdf:type owl:ObjectProperty ;
 	rdfs:range ww:Item ;
 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .
 	
-ww:hasIdentifier rdf:type owl:ObjectProperty ;
-	rdfs:label "hasIdentifier"@en ;
-	rdfs:comment "Relates the work to an identifier that is preferred, either a unique system-generated identifier that governs interaction between systems, or one used for human citation, such as a manuscript number to be cited in footnotes."@en ;
+ww:hasIdentifier rdf:type owl:ObjectProperty ;  
+	rdfs:label "hasIdentifier"@en ;  
+	rdfs:comment "Relates the item to a unique system-generated identifier that governs interaction between systems and is regarded as canonical within the Wellcome data ecosystem."@en ;   rdfs:domain ww:Item ;  rdfs:range ww:Identifier ;  rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .     ww:hasSourceIdentifier rdf:type owl:ObjectProperty ;  rdfs:label "hasSourceIdentifier"@en ;  rdfs:comment "Relates an item to an identifier used within one of the Wellcome Library's databases or other systems to refer to works or items within the Library's holdings, whether for internal technical purposes or for human citation."@  rdfs:domain ww:Item  rdfs:range ww:SourceIdentifier  rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/works> .    ww:hasSourceIdentifierAuthority rdf:type owl:ObjectProperty ;  rdfs:label "hasSourceIdentifierAuthority" @en ;  rdfs:comment "Relates a SourceIdentifier to a particular authoritative source of such identifiers: for example, if the SourceIdentifier is MS.49 this property might indicate that this identifier has its origins in the Wellcome Library's CALM archive management system."@en ;  
```

Comment:
Thanks - yes, cut-and-paste from my Notepad++ resulted in a slab of stuff coming in without line endings so it ended up hiding at the end of the line, thanks for spotting that!